src/Indicator.vala

```diff
@@ -210,6 +219,30 @@ public class Notifications.Indicator : Wingpanel.Indicator {
             warning ("Failed to open notifications settings: %s", e.message);
         }
     }
+
+    private void update_tooltip () {
+        uint number_of_notifications = Session.get_instance ().get_session_notifications ().length ();
+        string accel_label;
+
+        if (notify_settings.get_boolean ("do-not-disturb")) {
+            accel_label = """<span weight="600" size="smaller" alpha="75%">Middle-click to disable Do Not Disturb</span>""";
+        } else {
+            accel_label = """<span weight="600" size="smaller" alpha="75%">Middle-click to enable Do Not Disturb</span>""";
+        }
```

Comment:
for localization, and just to be more DRY, the formatting should be split from the string, so you want something like:
```suggestion
        if (notify_settings.get_boolean ("do-not-disturb")) {
            accel_label = _("Middle-click to disable Do Not Disturb");
        } else {
            accel_label = _("Middle-click to enable Do Not Disturb");
        }

        accel_label = """<span weight="600" size="smaller" alpha="75%">%s</span>""".printf (accel_label);
```

Reply:
Got it! Would it make sense to swap out the quotation marks for styling to single quotation marks? i.e. 

```vala
dynamic_icon.tooltip_markup = "%s\n<span weight='600' size='smaller' alpha='75%'>%s</span>".printf (description, accel_label)
```

That way, the newline character doesn't get ignored since the string uses triple quotations `"""`.

Reply:
Actually, it's fine. I'll just update `accel_label` with its styling and join them up later with `"%s\n%s".printf (description, accel_label` instead.